### PR TITLE
Feat swift large file

### PR DESCRIPTION
**Pull request by abnegate**

### CODE REVIEW

File: templates/swift/Sources/Client.swift.twig [hunk 1]
```diff
@@ -342,6 +345,14 @@ open class Client {
                 default:
                     var message = ""
 
+                    if response.body == nil {
+                        completion(.failure(AppwriteError(
```
Comment (lohanidamodar) [inline templates/swift/Sources/Client.swift.twig:349]: Remove Hardcoded Appwrite

File: templates/swift/Sources/Client.swift.twig [hunk 2]
```diff
@@ -363,7 +374,77 @@ open class Client {
...
 
-    private func randomBoundary() -> String {
+    func chunkedUpload<T>(
+        path: String,
+        headers: inout [String: String],
+        params: inout [String: Any?],
+        paramName: String,
+        convert: (([String: Any]) -> T)? = nil,
+        onProgress: ((Double) -> Void)? = nil,
+        completion: ((Result<T, AppwriteError>) -> Void)? = nil
```
Comment (lohanidamodar) [inline templates/swift/Sources/Client.swift.twig:384]: Remove hardcoded appwrite

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
Great work @abnegate, just few comments.

File: tests/SDKTest.php [hunk 3]
```diff
@@ -163,12 +163,13 @@ class SDKTest extends TestCase
                 'cp tests/languages/swift-server/Tests.swift tests/sdks/swift-server/Tests/AppwriteTests/Tests.swift',
             ],
             'envs' => [
-                'swift-5.5' => 'docker run --rm -v $(pwd):/app -w /app/tests/sdks/swift-server swift:5.5 swift test',
+                'swift-5.5' => 'docker run --rm -v $(pwd):/app -w /app/tests/sdks/swift-server swiftarm/swift:5.5.2-focal-multi-arch swift test',
```
[the remark this answers is not in the document]
Comment (abnegate) [reply tests/SDKTest.php:166]: @lohanidamodar This image supports arm64 as well as amd64 so tests can also run locally on M1 Macs

**Review by lohanidamodar — APPROVED** [review]
Looks good to me.

File: templates/swift/Sources/Client.swift.twig [hunk 4]
```diff
@@ -363,7 +374,77 @@ open class Client {
...
+                case let .failure(error):
+                    completion?(.failure(error))
+                    return
+                }
+            }
+            
+            group.wait()
+
+            offset += Client.chunkSize
+            headers["x-appwrite-id"] = result["$id"] as? String
```
Comment (christyjacob4) [inline templates/swift/Sources/Client.swift.twig:439]: If you could replace this string with the value from the spec, it would be great 👍

**Review by christyjacob4 — APPROVED** [review]
Just a minor suggestion

**Pull request merged**